src/emulator/functionsEmulator.ts

```diff
@@ -113,41 +92,34 @@ export class FunctionsEmulator implements EmulatorInstance {
     });
 
     hub.get("/", async (req, res) => {
-      res.send(
-        JSON.stringify(
-          await getTriggersFromDirectory(this.projectId, this.functionsDir, this.firebaseConfig),
-          null,
-          2
-        )
-      );
+      res.json({ status: "alive" });
     });
 
     // The URL for the function that the other emulators (Firestore, etc) use.
     // TODO(abehaskins): Make the other emulators use the route below and remove this.
-    const internalRoute = "/functions/projects/:project_id/triggers/:trigger_name";
+    const backgroundFunctionRoute = "/functions/projects/:project_id/triggers/:trigger_name";
 
     // The URL that the developer sees, this is the same URL that the legacy emulator used.
-    const externalRoute = `/:project_id/:region/:trigger_name`;
+    const httpsFunctionRoute = `/:project_id/:region/:trigger_name`;
 
     // A trigger named "foo" needs to respond at "foo" as well as "foo/*" but not "fooBar".
-    const functionRoutes = [
-      internalRoute,
-      `${internalRoute}/*`,
-      externalRoute,
-      `${externalRoute}/*`,
-    ];
+    const httpsFunctionRoutes = [httpsFunctionRoute, `${httpsFunctionRoute}/*`];
```

Comment:
Just want to make sure that callable functions get this treatment, not just "normal" HTTPS functions.  Can you confirm (or maybe even add a test)?

Reply:
Good point good point, should add a test

Reply:
Confirmed callable still work, will look at adding a test, but prolly not for this PR